python/lsst/ts/ATHexapod/controller.py

```diff
@@ -97,13 +98,13 @@ def is_connected(self):
             Command to send to hexapod. A 'newline' character will be appended.
         has_response : `bool`
             Does the command have a response?
-        multi_line : `bool`
-            Is the response received in multiple lines?
+        num_line : `int`
+            The number of expected lines.
```

Comment:
Is this also the number of expected replies? I hope it is. Please say so in this doc string.